app/modules/api/user.py

```diff
@@ -71,4 +73,31 @@ def join_team_token(json_content):
             "email": g.user.email,
             "team_id": g.user.team.id,
         }
-    )
\ No newline at end of file
+    )
+
+
+@bp.route('/register_hours', methods=['POST'])
+@req_helper.api_check_json("project_id", "iso_week", "hours")
```

Comment:
As i mentioned i still have no idea how to implement the schedule thing on the front, maybe passing the iso_week is not the easiest option... unless you add some api that can give me the isoweek for a given date, or better yet the ordinal number, that way i can ask the user to pick a start date and # of weeks of duration, hours for all those weeks and send the ordinals (because i only need to add 1 to that number for each week)

As an extra idea for later probably an /append_week_hours would be nice, so that if a project gets extended it automatically adds to the subsequent week... but maybe not for mvp.